### DISCUSSION TITLE
Hard-stop billing for projects.

### DISCUSSION THREAD

**Discussion by itsmeadarsh2008** [vercel/community#5952] [Idea]
I understand your concern regarding the billing for projects. To prevent excessive credit consumption and potential DDoS attacks, it is crucial to implement a hard-stop billing system. This will ensure that the bill does not inflate beyond a certain limit, which will help control the costs and prevent any unwanted surprises. I suggest discussing this with your technical team to come up with an effective implementation plan.

**Comment by leerob** [comment]
You can pause deployments using our Spend Management functionality: https://vercel.com/blog/introducing-spend-management-realtime-usage-alerts-sms-notifications

We're working on making it easier to pause projects automatically, if you prefer that approach over using a webhook, as well.
